google-cloud-datastore/lib/google/cloud/datastore/dataset/aggregate_query_results.rb

```diff
@@ -57,9 +57,24 @@ class AggregateQueryResults
           # @return [Google::Protobuf::Timestamp]
           attr_reader :read_time
 
+          # Query explain metrics. This is only present when the
+          # [RunQueryRequest.explain_options][google.datastore.v1.RunQueryRequest.explain_options]
```

Comment:
Please use yardoc format for cross-reference links in handwritten code. e.g.

    {Google::Cloud::Datastore::V1::RunQueryRequest#explain_options RunQueryRequest.explain_options}

Yardoc itself doesn't know how to interpret protobuf cross-references. (When these show up in proto docstrings, the GAPIC generator converts them to yardoc format.)